### PR TITLE
create catalog

### PR DESCRIPTION
**Pull request by andrewchornobai**

### CODE REVIEW

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
add your demo and test-report links, please

**Comment by andrewchornobai** [comment]
Links in readme file didn't works or what?

**Comment by andrewchornobai** [comment]
Links in readme file didn't works or what?

**Comment by andrewchornobai** [comment]
demo - https://andrewchornobai.github.io/layout_catalog/
 test-report - https://andrewchornobai.github.io/layout_catalog/report/html_report/

**Comment by andrewchornobai** [comment]
i think the link don't work, maybe it's wrong path in README file by DEMO LINK and TEST REPORT LINK

**Comment by andrewchornobai** [comment]
PR - https://github.com/mate-academy/layout_catalog/compare/master...andrewchornobai:layout_catalog:develop

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Nothing should be broken if I add more cards 
image

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,215 @@
-body {
+* {
   margin: 0;
+  padding: 0;
```
Comment (SanyaBratashchuk) [inline src/styles/main.scss:3]: Do not apply any rules to the `*` selector except `box-sizing`

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,215 @@
...
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.header {
```
Comment (SanyaBratashchuk) [inline src/styles/main.scss:13]: move each BEM block to the separate file

Comment (SanyaBratashchuk) [reply src/styles/main.scss:13]: FIx it everywhere

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,215 @@
...
+    background-image: url(../images/star-active.svg);
+  }
+  
+  &__stars--4 &__item:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+  
+  &__stars--5 &__item:nth-child(-n + 5) {
+    background-image: url(../images/star-active.svg);
+  }
```
Comment (SanyaBratashchuk) [inline src/styles/main.scss:167]: use scss loop

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 4]
```diff
@@ -4,9 +4,403 @@
...
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img 
+        class="header__moyoLogo" 
+        src="images/logo.png" 
+        alt="moyo_logo"
+        >
```
Comment (ViktorHavryliuk) [inline src/index.html:19]:
```suggestion
        <img 
          class="header__moyoLogo" 
          src="images/logo.png" 
          alt="moyo_logo"
        >
```

File: src/index.html [hunk 5]
```diff
@@ -4,9 +4,403 @@
...
+      </a>
+      <nav class="header__nav">
+        <ul class="header__linksContainer">
+          <li class="header__liClass">
+            <a 
+              class="header__links is-active" 
+              href="#"
+            >Apple</a>
```
Comment (ViktorHavryliuk) [inline src/index.html:27]:
```suggestion
            <a 
              class="header__links is-active" 
              href="#"
            >
              Apple
            </a>
```

File: src/styles/blocks/card.scss [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  &__info {
+    padding-bottom: 16px;
+  }
+
+  &__text-name {
+    font-weight: 500;
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
```
Comment (ViktorHavryliuk) [inline src/styles/blocks/card.scss:24]: move commonly used colors to variables

**Review by ViktorHavryliuk — CHANGES_REQUESTED** [review]
Add some separation between sibling html tags (checklist p.2):
image

File: src/index.html [hunk 7]
```diff
@@ -4,9 +4,405 @@
...
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img 
+          class="header__moyoLogo" 
+          src="images/logo.png" 
+          alt="moyo_logo"
+        >
+      </a>
+      <nav class="header__nav">
```
Comment (DarkMistyRoom) [inline src/index.html:21]: Add empty lines between multiline sibling blocks of HTML

File: src/index.html [hunk 8]
```diff
@@ -4,9 +4,405 @@
...
+        <ul class="header__linksContainer">
+          <li class="header__liClass">
+            <a 
+              class="header__links is-active" 
+              href="#"
+            >
+            Apple
+            </a>
```
Comment (DarkMistyRoom) [inline src/index.html:29]:
```suggestion
            >
              Apple
            </a>
```

File: src/index.html [hunk 9]
```diff
@@ -4,9 +4,405 @@
...
+            <div class="card__product-code">Product code: 195434</div>
+          </div>
+          <div class="card__stars-text">
+            <div class="card__stars card__stars--4">
+              <span class="card__item"></span>
+              <span class="card__item"></span>
+              <span class="card__item"></span>
+              <span class="card__item"></span>
+              <span class="card__item"></span>
+            </div>
```
[the remark this answers is not in the document]
Comment (DarkMistyRoom) [reply src/index.html:83]: you shouldn't make it an element of `card`. just reuse the whole block

File: src/index.html [hunk 10]
```diff
@@ -4,9 +4,405 @@
...
+            <div class="card__text-revies">Reviews: 5</div>
```
[the remark this answers is not in the document]
Comment (DarkMistyRoom) [reply src/index.html:84]: fix it for all text elements

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 9 above)
[the remark this answers is not in the document]
Comment (andrewchornobai) [reply src/index.html:83]: image
image

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
fix previous comments plz, and if you have questions - ask them in the chat

**Comment by andrewchornobai** [comment]
> fix previous comments plz, and if you have questions - ask them in t

**Pull request closed without merging**